### ISSUE TITLE
Log warning line numbers for included files

### ISSUE THREAD

**Issue by hendrikbl** [thomasjo/atom-latex#430]
I'm using `\include` a lot to structure my project. When there is a warning in an included file the log only shows the position where the file is included in the root file. Is there any way of logging the actual position of the warning so i dont have to look through the entire file I included?

**Comment by yitzchak** [comment]
Thanks for reporting the issue!

If a line number is reported in a warning message then it should be included in the reference from the log panel. That said, it doesn't always work because log messages produced by TeX are incredibly varied.

If you a simple document that demonstrates the issue that help enormously.

**Comment by hendrikbl** [comment]
Thanks for the quick response! 

Here's an example:

### main.tex
This is the file with the include in it
```latex
\begin{document}
\frontmatter
\tableofcontents
\mainmatter
\include{include}
\end{document}
```

### include.tex
This is the file with an underfull hbox
```latex
\begin{titlepage}
	\dotfill\\
	Prof. XYZ\\
	Department of XYZ\\
	University of whatever, \\
	City, Country\\

	Date: ............\\
\end{titlepage}
```

But the log says the warning is in `main.tex (2-7)` 
![warning message](https://user-images.githubusercontent.com/9558391/31768353-52e15f0e-b4cf-11e7-9f27-4c3e5ef7c6b3.PNG)

I'm not sure if thats something you can fix though or if it's on TeX related.

**Comment by yitzchak** [comment]
Thanks for sample document. I have reproduced the issue.

It looks like TeX is putting a file reference on an earlier line (in the log) before the group of box messages. Not sure if it consistent in that yet. Fixing it will require enhancing the log parser so it will take a bit work to resolve.

**Comment by hendrikbl** [comment]
Okay, I already guessed it isn't something that can be fixed easily...

Thanks for looking into it!

**Comment by yitzchak** [comment]
This enhancement will be be in the next version. We just merged the code for the `latexmk` based builder and support for DiCy is in the works.

**Comment by yitzchak** [comment]
Added in both builders and released in v0.49.0
